<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 229, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tituloLogin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Geometr212 BkCn BT</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Geometr212 BkCn BT&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tituloUsuario">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Geometr212 BkCn BT&quot;;
</string>
    </property>
    <property name="text">
     <string>Usu√°rio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tituloSenha">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Geometr212 BkCn BT&quot;;</string>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoAcessar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
font: 13pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Acessar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usuario">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="senha">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoCadastrar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>581</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
font: 13pt &quot;Nirmala UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
